modules/process/manager.go

```diff
@@ -44,18 +46,27 @@ type Manager struct {
 
 	counter   int64
 	processes map[int64]*Process
+
+	// guid is a unique id for a gitea instance
+	guid int64
 }
 
 // GetManager returns a Manager and initializes one as singleton if there's none yet
 func GetManager() *Manager {
 	if manager == nil {
 		manager = &Manager{
 			processes: make(map[int64]*Process),
+			guid:      int64(gouuid.New().ID()),
 		}
 	}
 	return manager
 }
 
+// GUID return a unique id of a gitea instance
+func (pm *Manager) GUID() int64 {
+	return pm.guid
```

Comment:
The guid will be changed once the Gitea restarted?

Reply:
yes